src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      data-qa="big"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form first_form"
+    >
+      <div class="input__wrapper__big">
+        <img src="./images/Search.svg" alt="Icon search">
+        <input
+          class="large"
```

Comment:
[checklist p.2](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

Here you need to set one class and write all styles to it. In the small form, you need to add two classes. For example, this input would have class `input` and the small input would have classes `input input__small`. So you just once would set styles to `input` and then you would add all styles that are different for the small input into `input__small`.